phpstan.neon

```diff
@@ -1,22 +1,32 @@
 parameters:
+    level: 8
+    paths:
+        - src/
+    bootstrapFiles:
+        - tests/bootstrap.php
     checkGenericClassInNonGenericObjectType: false
     checkMissingIterableValueType: false
     ignoreErrors:
-        - '#.+Generated\\Shared\\Transfer.+#'
-        - '#Parameter .+ of method .+ has invalid typehint type Orm\\Zed\\Payone\\Persistence\\.+#'
-        - '#Return typehint of method .+ has invalid type Orm\\Zed\\Payone\\Persistence\\.+#'
+        - '#Expression on left side of \?\? is not nullable\.#'
+        - '#Parameter .+ of method .+ has invalid type .+#'
+        - '#Method .+ has invalid return type .+#'
+        - '#Method .+ should return .+ but returns .+#'
```

Comment:
Those are too generic, they match everything and are not valid as such.

Reply:
Fixed.